web-admin/src/routes/-/auth/success/+page.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+  import RillLogoSquareNegative from "@rilldata/web-common/components/icons/RillLogoSquareNegative.svelte";
+  import CtaContentContainer from "@rilldata/web-common/components/calls-to-action/CTAContentContainer.svelte";
+  import CtaHeader from "@rilldata/web-common/components/calls-to-action/CTAHeader.svelte";
+  import CtaLayoutContainer from "@rilldata/web-common/components/calls-to-action/CTALayoutContainer.svelte";
+  import CtaMessage from "@rilldata/web-common/components/calls-to-action/CTAMessage.svelte";
+</script>
+
+<svelte:head>
+  <title>Authorised to Rill successfully</title>
+</svelte:head>
+
+<CtaLayoutContainer>
+  <CtaContentContainer>
+    <RillLogoSquareNegative size="84px" />
+    <CtaHeader>Authorised to Rill successfully</CtaHeader>
+    <CtaMessage>Close this page and continue in the Rill CLI.</CtaMessage>
```

Comment:
```suggestion
    <CtaMessage>Close this page and continue setup in the Rill CLI.</CtaMessage>
```

Reply:
I intentionally removed `setup` here to make it more generic since there is no setup to be done in `rill login`.